starter_code/index.html

```diff
@@ -15,12 +15,88 @@
+			<div class="blog">
+				<div class="main">
+					<div class="top">
+						<div class="header">
+							<h1>How I implemented Relaxr in 2 <br> weeks and changed my life.</h1>
+						</div>
+						<div class="image-top"></div>
+						<div class="text">
+							<div class="first-letter">L</div>
+							orem Ipsum is simply dummy text of the printing and typesetting industry. 
+							Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+							when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+							It has survived not only five centuries, but also the leap into electronic typesetting, 
+							remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset 
+							sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like 
+							Aldus PageMaker including versions of Lorem Ipsum. <br><br>
+							<div class="read-more">Read more ></div> <br>
+							<hr>
+						</div>
+					</div>
+					<div class="bottom">
+						<div class="header">
+							<h1>I traveled across the globe thanks to Relaxr’s Excel automation</h1>
+						</div>
+						<div class="image-bottom"></div>
+						<div class="text">
+							<div class="first-letter">L</div>
+							orem Ipsum is simply dummy text of the printing and typesetting industry. 
+							Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+							when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+							It has survived not only five centuries, but also the leap into electronic typesetting, 
+							remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset 
+							sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like 
+							Aldus PageMaker including versions of Lorem Ipsum. <br><br> 
+							<div class="read-more">Read more ></div> <br>
+							<hr>
+						</div>
+					</div>
+				</div>
+				<div class="side">
+					<div class="categories">
+						<ul>Categories
+							<li>Success stories</li>
+							<li>Stats</li>
+							<li>How tos</li>
+							<li>Best of</li>
+							<li>Business tips</li>
+						</ul>
+					</div>
+					<div class="about">
+						<h1>About Relaxr</h1>
+						<p>
+							roin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.
+						</p>
+					</div>
+					<div class="ad-unit">Ad Unit</div>
+				</div>
+			</div>
+			<div class="sign-up">
+				<button>Sign Up Now</button>
+			</div>
 			<footer>
-				&copy; Footer Goes Here
-			</footer>
+					<h1>Relaxr</h1>
+					<div class="images">
+						<div class="twitter">
+							<img src="images/twitter.png">
+						</div>
+						<div class="facebook">
+							<img src="images/facebook.png">
+						</div>
+					</div>
+					<p>Copyright 2015. Relaxr.</p>
+				</footer> 				
```

Comment:
@Chaimberger I would fix the footer indentation, here, too. BUT overall you're doing well!